### PR TITLE
Set up Dependabot for gomod and github-actions dependencies

### PR DESCRIPTION
**Pull request by mmorel-35**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: weekly
```
Comment (vmihailenco) [inline .github/dependabot.yml:6]: Weekly is probably rare enough, but we can consider increasing this interval to monthly to lower the noise created by dependabot commits.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: weekly
```
Comment (vmihailenco) [inline .github/dependabot.yml:10]: So this updates the versions of Github actions itself, right? Interesting...

**Review by vmihailenco — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (mmorel-35) [reply .github/dependabot.yml:10]: Exactly! You can learn more about this [here](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot)

File: .github/dependabot.yml (hunk 1 above)
Comment (mmorel-35) [reply .github/dependabot.yml:6]: I doubt there will be much noise as the number of dependencies is quite low

**Review by monkey92t — APPROVED** [review]

**Pull request merged**